nfs.yml

```diff
@@ -1,7 +1,28 @@
 ---
 # This playbook uses the Ansible OpenStack modules to create a cluster
 # using a number of baremetal compute node instances, and configure it
-# for a NFS server 
+# for a NFS server.
+
+# Prevent nfs-server from failing to start after reboot: IPv6 is disabled via
+# sysctl in the JASMIN CentOS 7 image, but initramfs doesn't include this
+# sysctl setting. This prevents rpcbind (on which nfs-server depends) from
+# starting on boot. See https://access.redhat.com/solutions/2798411 for more
+# details. Unfortunately the Red Hat solution is incomplete, as dracut doesn't
+# include /etc/sysctl* by default, so we modify dracut.conf before rebuilding.
+- hosts:
+    - nfs_servers
+  become: true
+  tasks:
+    - name: Add sysctl configuration to dracut.conf
+      blockinfile:
+        path: /etc/dracut.conf
+        block: |
```

Comment:
Are you preferring `blockinfile` over `lineinfile` because then we get the delimiters `# BEGIN ANSIBLE MANAGED CONTENT` ?

Reply:
I actually went with `blockinfile` because I had used it recently :) Happy to switch to `lineinfile`.